src/transaction/log_page_buffer.c

```diff
@@ -7352,6 +7352,11 @@ logpb_checkpoint_trantable (THREAD_ENTRY * const thread_p)
     LOG_LSA dummy_smallest_tran_lsa = NULL_LSA;
     trantable_checkpoint_info.load_trantable_snapshot (thread_p, dummy_smallest_tran_lsa);
 
+    // Append to log
+    log_append_trantable_snapshot (thread_p, trantable_checkpoint_info);
```

Comment:
A comment would be useful that the trantable is saved to metafile and also added to log and for which purpose for each.
I assume that, for the long term, it can be done without writing it to metafile and, instead, recovering it from the log - maybe during a supplemental initialization step (instead of, or alongside recovery analysis).
Am I correct?

Reply:
Correct. I updated the comment and added a todo.